libexec/klab-prove

```diff
@@ -105,6 +105,13 @@ concrete_rules=(
   EVM-TYPES.#asByteStackAux.recursive
   EVM-TYPES.#padToWidth
   EVM.#memoryUsageUpdate.some
+  # EVM.Cgascap
```

Comment:
I don't think these comments should be added

Reply:
oops